e2e_tests/account_test.py

```diff
@@ -0,0 +1,48 @@
+import os
+import unittest
+from datetime import datetime, date, timedelta
+from api.unit import Unit
+from models.account import *
+
+
+class AccountE2eTests(unittest.TestCase):
+    token = os.environ.get("token")
+    client = Unit("https://api.s.unit.sh", token)
+
+    def test_create_deposit_account(self):
```

Comment:
add another test for Joint deposit account...

in order to create one like this you should create a depositAccount with multiple owners (use customers relationship instead of customer)